include/bl_provision.h

```diff
@@ -4,21 +4,19 @@
  * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
  */
 
-#ifndef PROVISION_H_
-#define PROVISION_H_
+#ifndef BL_PROVISION_H_
+#define BL_PROVISION_H_
 
-/*
- * Copyright (c) 2018 Nordic Semiconductor ASA
- *
- * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
- */
+#ifdef __cplusplus
+extern "C" {
+#endif
 
 #include <zephyr/types.h>
 #include <string.h>
 
-#ifdef __cplusplus
-extern "C" {
```

Comment:
Don't we need this? Ref the check of __cplusplus at the end of the file.

Reply:
Fixed.